### PR TITLE
Minor burn fixes

### PR DESCRIPTION
**Pull request by bachstatter**
- Dont display unstaking note when you're above target
- Disable burn to target when already above
- better tooltips
Screen Shot 2022-11-07 at 4 47 17 pm

### CODE REVIEW

File: v2/components/Burn/Burn.tsx [hunk 1]
```diff
@@ -249,15 +251,24 @@ export const BurnUi = ({
             </Badge>
             <Badge
               variant="burn"
-              sx={{
-                bg: activeBadge === 'toTarget' ? 'cyan.500' : 'whiteAlpha.300',
-                color: activeBadge === 'toTarget' ? 'black' : 'cyan.500',
+              bg={activeBadge === 'toTarget' ? 'cyan.500' : 'whiteAlpha.300'}
+              color={activeBadge === 'toTarget' ? 'black' : 'cyan.500'}
+              cursor={isAboveTarget ? 'not-allowed' : 'pointer'}
```
Comment (jmzwar) [inline v2/components/Burn/Burn.tsx:256]: nice.

File: v2/components/Burn/Burn.tsx [hunk 2]
```diff
@@ -249,15 +251,24 @@ export const BurnUi = ({
...
+              _hover={{
+                cursor: isAboveTarget ? 'not-allowed' : 'pointer',
               }}
               ml={1}
-              onClick={() => handleBadgePress('toTarget')}
+              onClick={isAboveTarget ? undefined : () => handleBadgePress('toTarget')}
```
Comment (jmzwar) [inline v2/components/Burn/Burn.tsx:261]: () => {} ?

Man I hate undefined :P

**Review by jmzwar — APPROVED** [review]
I think good to merge

File: v2/components/Burn/Burn.tsx (hunk 2 above)
Comment (bachstatter) [reply v2/components/Burn/Burn.tsx:261]: haha undefined is a valid prop, the component could behave differently if onClick is defined or not.. Guess it prob doesn't in this case, but still

**Pull request merged**